src/cljs/main/broadfcui/page/profile.cljs

```diff
@@ -169,6 +174,13 @@
      [:div {:style {:clear "both" :margin "0.5em 0"}}
       [:label {}
        (style/create-form-label label)
+       #_(this :render-text-field key true
```

Comment:
did you mean to remove this?

Reply:
Ah, thanks. This and some other stuff left over from an abandoned refactoring. I'll remove it.